core/modules/simpletest/tests/session.test

```diff
@@ -464,6 +464,56 @@ class SessionHttpsTestCase extends BackdropWebTestCase {
   }
 
   /**
+   * Tests that empty session IDs do not cause unrelated sessions to load.
+   */
+  public function testEmptySessionId() {
+    global $is_https;
+
+    if ($is_https) {
+      $secure_session_name = session_name();
+    }
+    else {
+      $secure_session_name = 'S' . session_name();
+    }
+
+    // Enable mixed mode for HTTP and HTTPS.
+    variable_set('https', TRUE);
```

Comment:
This line (and the comment) can be removed. Backdrop no longer uses variable_set() and the equivalent of setting this property is added for us in session_test.module's `session_test_boot()` function.